contracts/POA/ConquestEntriesFactory.sol

```diff
@@ -0,0 +1,147 @@
+   * @param _skyweaverAssetsAddress The address of the ERC-1155 Assets Token contract
+   * @param _conquestEntryTokenId   Conquest entry token id
+   * @param _silverCardsRange       ID range that includes only silver cards
+   */
+  constructor(
+    address _skyweaverAssetsAddress,
+    uint256 _conquestEntryTokenId,
+    Range memory _silverCardsRange
+  ) public 
+  {
+    require(
+      _skyweaverAssetsAddress != address(0),
+      "Conquest#constructor: INVALID_INPUT"
+    );
+
+    // Assets
+    skyweaverAssets = ISkyweaverAssets(_skyweaverAssetsAddress);
+    conquestEntryID = _conquestEntryTokenId;
+    silverRange = _silverCardsRange;
+  }
+
+  
+  /***********************************|
+  |      Receiver Method Handler      |
+  |__________________________________*/
+
+  /**
+   * @notice Prevents receiving Ether or calls to unsuported methods
+   */
+  fallback () external {
+    revert("ConquestEntryFactory#_: UNSUPPORTED_METHOD");
+  }
+
+  /**
+   * @notice Players converting silver cards to conquest entries
+   * @dev Payload is passed to and verified by onERC1155BatchReceived()
+   */
+  function onERC1155Received(
+    address _operator,
+    address _from,
+    uint256 _id, 
+    uint256 _amount, 
+    bytes memory _data
+  )
+    public override returns(bytes4)
+  {
+    // Convert payload to arrays to pass to onERC1155BatchReceived()
+    uint256[] memory ids = new uint256[](1);
+    uint256[] memory amounts = new uint256[](1);
+    ids[0] = _id;
+    amounts[0] = _amount;
+
+    // Will revert call if doesn't pass
+    onERC1155BatchReceived(_operator, _from, ids, amounts, _data);
+    
+    // Return success
+    return IERC1155TokenReceiver.onERC1155Received.selector;
+  }
+
+  /**
+   * @notice Players converting silver cards to conquest entries
+   * @param _from    Address who sent the token
+   * @param _ids     An array containing ids of each Token being transferred
+   * @param _amounts An array containing amounts of each Token being transferred
+   */
+  function onERC1155BatchReceived(
+    address, // _operator
+    address _from,
+    uint256[] memory _ids,
+    uint256[] memory _amounts,
+    bytes memory // _data
+  )
+    public override returns(bytes4)
+  {
+    require(msg.sender == address(skyweaverAssets), "ConquestEntryFactory#onERC1155BatchReceived: INVALID_TOKEN_ADDRESS");
+    uint256 n_entries = 0; // Number of entries to mint
+
+    // Validate IDs and count number of entries to mint
+    for (uint256 i = 0; i < _ids.length; i++) {
+      require(
+        silverRange.min <= _ids[i] && _ids[i] <= silverRange.max, 
+        "ConquestEntryFactory#onERC1155BatchReceived: ID_IS_OUT_OF_RANGE"
+      );
+      n_entries = n_entries.add(_amounts[i]);
+    }
+
+    // Burn silver cards received
+    skyweaverAssets.batchBurn(_ids, _amounts);
+
+    // Mint conquest entries (silvers have 2 decimals)
+    skyweaverAssets.mint(_from, conquestEntryID, n_entries / 100, "");
```

Comment:
Maybe we should put the number of decimals that silver cards have into a constant.